saleor/payment/utils.py

```diff
@@ -495,3 +511,25 @@ def gateway_refund(payment, amount: Decimal = None) -> Transaction:
 
     _gateway_postprocess(transaction, payment)
     return transaction
+
+
+GATEWAYS_LABEL = "gateways"
+
+
+def extract_id_for_payment_gateway(user, gateway):
+    gateway_meta = user.get_private_meta(label=GATEWAYS_LABEL)
```

Comment:
How about:
```
    def get_private_meta(self, label):
         if label not in self.private_meta:
             return {}
         return self.private_meta[label]
OR 
    def get_private_meta(self, label):
         return self.private_meta.get(label, {})
```
And then you don't need to use so many `if`s
```
    gateway_config = gateway_meta.get(gateway)
    if not gateway_config;
        None
    return gateway_config.get("customer_id")

Reply:
I kind of dislike passing valid but empty structures when no value is really found, but in this case it probably makes more sense and is more practical.
Changed.